stripe/v2/_event.py

```diff
@@ -132,6 +136,7 @@ def __init__(self, payload: str) -> None:
         self.id = parsed["id"]
         self.type = parsed["type"]
         self.created = parsed["created"]
+        self.livemode = parsed.get("livemode")
```

Comment:
we're missing reason; is that a blocker for this release?

Reply:
scroll scroll scroll a bit... gently down the page